CompilerSource/gcc_interface/gcc_backend.cpp

```diff
@@ -173,7 +173,9 @@ const char* establish_bearings(const char *compiler)
       }
       pos += idirstart.length();
     }
+  
     jdi::builtin->add_search_directory((enigma_root/"ENIGMAsystem/SHELL").u8string().c_str());
+    jdi::builtin->add_search_directory((enigma_root/"shared").u8string().c_str());
```

Comment:
Why does JDI need to look at `shared/` now?

Reply:
because engine includes headers from there

Reply:
Yeah but is it including `enigma_user` namespace stuff from here? If it's only including our internal stuff, I don't see why JDI needs to look at it.